source/features/select-notifications.tsx

```diff
@@ -131,9 +131,10 @@ const createDropdown = onetime(() => (
 		onToggle={resetFilters}
 	>
 		<summary
-			className="h6" // Match "Select all" style
-			data-hotkey="S"
+			className="h6 tooltipped tooltipped-n" // Match "Select all" style
```

Comment:
```suggestion
			className="h6 tooltipped tooltipped-n" // `h6` matches "Select all" style
```

Reply:
Ah, ok. With your suggested change it can obscure the top of the menu, but I guess as long as it is only one line it's only hiding the "Type" heading, so maybe that's okay?

<img width="550" alt="Screenshot 2024-08-22 at 08 34 20" src="https://github.com/user-attachments/assets/61390a4f-bf30-4880-98b5-76eced0f9231">